doc/classes/ParagraphLayout.xml

```diff
@@ -0,0 +1,583 @@
+			<argument index="3" name="color" type="Color">
+			<argument index="4" name="invisibles_color" type="Color">
+			</argument>
+			<description>
+				Draws the visual character cluster and returns rendered character cluster width.
+			</description>
+		</method>
+		<method name="draw_paragraph">
+			<return type="Rect2">
+			</return>
+			<argument index="0" name="canvas_item" type="RID">
+			<argument index="1" name="offset" type="Point2">
+			<argument index="2" name="color" type="Color">
+			<argument index="3" name="invisibles_color" type="Color">
+			<argument index="4" name="clip_rect" type="Rect2">
+			</argument>
+			<description>
+				Draws the paragraph and returns bounding box.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="base_direction" type="TextDirection" setter="set_base_direction" getter="get_base_direction">
+			Paragraph base reading direction (any of the [code]DIR_*[/code] values).
+		</member>
+		<member name="autowrap" type="bool" setter="set_autowrap" getter="get_autowrap">
+			Autowrap.
+		</member>
+		<member name="max_width" type="int" setter="set_max_width" getter="get_max_width">
+			Paragraph maximum line width.
+		</member>
+		<member name="line_spacing" type="float" setter="set_line_spacing" getter="get_line_spacing">
+			Paragraph line spacing multiplier.
+		</member>
+		<member name="align" type="ParagraphAlign" setter="set_align" getter="get_align">
+			Horizontal alignment mode.
+		</member>
+		<member name="tab_stops" type="Vector<int>" setter="set_tab_stops" getter="get_tab_stops">
+			Repeated pattern of tab stops, in em units.
+		</member>
+		<member name="show_invisibles" type="bool" setter="set_show_invisibles" getter="get_show_invisibles">
+			Invisible characters placeholder rendering mode.
+		</member>
+		<member name="kasida_to_space_ratio" type="float" setter="set_kasida_to_space_ratio" getter="get_kasida_to_space_ratio">
+			Amount of justification given to kashidas in ratio to spaces.
+		</member>
+		<member name="enable_shaping" type="bool" setter="set_enable_shaping" getter="get_enable_shaping">
+			Complex text shaping mode.
+		</member>
+	</members>
+	<constants>
+		<constant name="PARA_ALIGN_START" value="0">
+			Align paragraph lines to the trailing edge (default).
+		</constant>
+		<constant name="PARA_ALIGN_END" value="1">
+			Align paragraph lines to the leading edge.
+		</constant>
+		<constant name="PARA_ALIGN_LEFT" value="2">
+			Align paragraph lines to the left edge.
+		</constant>
+		<constant name="PARA_ALIGN_RIGHT" value="3">
+			Align paragraph lines to the right edge.
+		</constant>
+		<constant name="PARA_ALIGN_CENTER" value="4">
+			Align paragraph lines centered.
+		</constant>
+		<constant name="PARA_ALIGN_FILL" value="5">
+			Expand lines to fit the width, the last line is start-aligned.
+		</constant>
+		<constant name="PARA_ALIGN_FILL_ALL" value="6">
+			Expand all lines to fit the width.
+		</constant>
```

Comment:
CSS has a separate property for aligning last line (`text-align-last`), it would be nice to have something similar here. That would make `PARA_ALIGN_FILL_ALL` redundant.